src/i18n/index.js

```diff
@@ -72,6 +72,8 @@
  */
 
 export {
+  createIntl,
+  intlShape,
```

Comment:
`intlShape` is already exported below from `'./lib`. In v3 of `react-intl`, `intlShape` was removed from the library so as part of the upgrade to v5, we provided own our copy of `intlShape` instead below so it doesn't break for consumers.

Reply:
noted!